OrbitCaptureClient/CaptureClient.cpp

```diff
@@ -147,6 +150,7 @@ bool CaptureClient::StopCapture() {
     ERROR(
         "WritesDone on Capture's gRPC stream failed: unable to finish the "
         "capture in orderly manner, initiating emergency stop");
+    capture_listener_->OnCaptureFailed(ErrorMessage("Error while stopping capture."));
```

Comment:
Use the same error message as in ERROR? And note that this function is called off the capture thread - we generally do not want to call listener outside of capture thread - maybe handle this in capture thread?

Reply:
Thanks. I moved that to the capture thread and used the same string here.